server/src/main/java/module-info.java

```diff
@@ -146,6 +146,7 @@
     exports org.elasticsearch.action.support.master;
     exports org.elasticsearch.action.support.master.info;
     exports org.elasticsearch.action.support.nodes;
+    exports org.elasticsearch.action.support.local;
```

Comment:
I moved `TransportLocalClusterStateAction` to its own `local` package - together with the new `LocalClusterStateRequest`.